_posts/newsletters/2022-01-31-newsletter.md

```diff
@@ -0,0 +1,205 @@
+
+Most developers will agree that testing is an important part of software development. But is testing of research software any different than testing of other types of software? Is testing in a research context different from testing in general software enterprises? And how and what do you test in the first place? What can we learn from each other and from practices elsewhere?
+
+ The call will discuss, and share knowledge and experiences, around questions such as why you would want to test and what the benefits are, what technologies are out there, and what processes RSEs do (and don’t) employ to test their software. No matter if you have never heard of testing or if you are an expert, come join us for an interesting discussion and to hear and share some "testimonies" of what has worked and not worked for others.
+
+We have also changed the way we organize community calls. We meet two weeks prior to each community call to decide on a topic and discuss how to structure the call. Anyone is welcome to join the preparation call on [Zoom](https://mit.zoom.us/j/93471607274?pwd=aXUzNzZsWElpWnc4N2NXaWxOcVNlQT09).
+
+<a name="podcast"></a>
+# **RSE Stories Podcast**
+
+The [Research Software Engineer Stories podcast](http://us-rse.org/rse-stories/) has two new episodes posted. Check them out below!
+
+RSE Stories is looking for a new co-host and other volunteers to help with production. You don’t need to commit to anything: start with one episode and go from there. Interested? Reach out to Vanessa via Slack.
+
+* Chase Million: [No Shortage of Ideas](http://us-rse.org/rse-stories/2022/chase-million/)
+* Tim Haines: [For the Love of Learning](http://us-rse.org/rse-stories/2021/tim-haines/)
+
+<a name="scipy"></a>
+# **US-RSE at SciPy 2022**
+
+US-RSE members plan to submit a proposal for a [Birds of a Feather session at SciPy 2022](https://www.scipy2022.scipy.org/bof-sessions) about US-RSE. This will be a moderated panel discussion. If you are planning to attend SciPy 2022 in July in Austin and interested in representing US-RSE on the panel, please fill out this form: [https://forms.gle/Mx4qW1zH518Ev2Cm6](https://forms.gle/Mx4qW1zH518Ev2Cm6)
+
+[Submissions are also open](https://www.scipy2022.scipy.org/participate) for talks, posters, and tutorials at SciPy 2022. Please consider submitting if you are working with Python in any capacity.
+
+<a name="sc22"></a>
+# **US-RSE at [SC22](https://sc22.supercomputing.org/)**
+
+We have reserved a 10’ x 10’ small booth space for [SC22](https://sc22.supercomputing.org) in November. Global health situations willing, we hope to be at location 4214 with some of our RSE friends from around the world. The US-RSE #sc22 Slack channel will be starting soon. We look forward to chatting and brainstorming about fun RSE uses for a cozy 100 square foot space, and maybe answering the vexing question of how many RSEs can fit in a telephone booth.
```

Comment:
```suggestion
We have reserved a 10’ x 10’ small booth space for [SC22](https://sc22.supercomputing.org) in November. Global health situations willing, we hope to be at location 4214 with some of our RSE friends from around the world. The US-RSE #sc22 Slack channel will be created soon. We look forward to chatting and brainstorming about fun RSE uses for a cozy 100 square foot space, and maybe answering the vexing question of how many RSEs can fit in a telephone booth.
```
and if it was already created, we could say "join us in the #sc22 slack channel."

Reply:
How many RSEs can you fit in a telephone booth hmm. Probably just one, lol. Especially if it's me, I have my space needs for stretching :t-rex: